parser.html

```diff
@@ -0,0 +1,262 @@
+<link rel="import" href="common.html">
+<link rel="import" href="lexer.html">
+<script>
+function ShadeyCssParser() {
+  'use strict';
+
+  var lexer = ShadeyCssLexer();
```

Comment:
Maybe use ES6 modules and compile to several target modules systems? I'd want AMD in Designer. Hydrolysis probably wants require(), and core would want globals.